src/cli.rs

```diff
@@ -302,6 +317,20 @@ impl Default for ImageCliOptions {
     }
 }
 
+impl Default for ConfigCliOptions {
+    fn default() -> Self {
+        ConfigCliOptions {
+            // Not sure about unwrap
+            config_path: Some(
+                dirs::config_dir()
+                    .expect("Could not find $HOME!")
```

Comment:
```suggestion
                    .expect("OS to have a home directory")
```
`$HOME` isn't used in Windows, and this can only fail on operating systems that don't have standard home paths AFAIK. Also, I switched to the style where `expect` is part of the sentence, though you don't have to follow that.